pkg/pxc/statefulset.go

```diff
@@ -168,6 +169,30 @@ func PodAffinity(af *api.PodAffinity, app api.App) *corev1.Affinity {
 	return nil
 }
 
+func PodTopologySpreadConstraints(tscs []corev1.TopologySpreadConstraint, ls map[string]string) []corev1.TopologySpreadConstraint {
+	result := make([]corev1.TopologySpreadConstraint, 0, len(tscs))
+
+	for _, tsc := range tscs {
+		if tsc.LabelSelector == nil && tsc.MatchLabelKeys == nil && len(ls) > 0 {
+			tsc.LabelSelector = &metav1.LabelSelector{
+				MatchLabels: ls,
+			}
+		}
+		if tsc.MaxSkew == 0 {
+			tsc.MaxSkew = 1
+		}
+		if tsc.TopologyKey == "" {
+			tsc.TopologyKey = api.DefaultAffinityTopologyKey
+		}
+		if tsc.WhenUnsatisfiable == "" {
+			tsc.WhenUnsatisfiable = corev1.DoNotSchedule
+		}
```

Comment:
I'm not sure about this as default. Maybe we need to use `ScheduleAnyway` as default. wdyt @spron-in @hors?

Reply:
Hey. Let's use `ScheduleAnyway`